infra/lib/attendant-ide/compute-construct.ts

```diff
@@ -54,8 +58,11 @@ export class ComputeConstruct extends Construct {
     userData.addCommands(
       // Add .NET repo
       `rpm -Uvh ${dotNetRepo}`,
+      `yum clean all`,
       // Install general dependencies
-      `yum install -y  ${osPackages.join(' ')}`,
+      `yum install -y --downloadonly  ${osPackages.join(' ')}`,
+      `sleep 20`,
+      `yum install -y ${osPackages.join(' ')}`,
```

Comment:
I have no idea why, but `yum` was finishing the transaction before creating the cache and crashing randomly.

Using this solution we first download packages and after we can install them.


```sh
Total                                            46 MB/s | 359 MB     00:07
[Errno 2] No such file or directory: '/var/cache/dnf/amazonlinux-87b6c41b372540cd/packages/langpacks-core-font-en-3.0-21.amzn2023.0.4.noarch.rpm'
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'yum clean packages'.
Redirecting to /bin/systemctl start docker.service
Failed to start docker.service: Unit docker.service not found.
```